doc/classes/EditorExport.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorExport" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Manages export platforms and presets for exporting Godot projects.
+	</brief_description>
+	<description>
+		Singleton class that handles the export system in Godot. It maintains collections of export platforms (like Windows, Linux, macOS, Android, iOS, Web) and export presets (configurations for each platform). This class is responsible for loading and saving export configurations, as well as providing access to export functionality through the editor.
+		Export platforms define how projects are exported to specific target platforms, while export presets store the configuration settings for each export target.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="add_export_platform">
+			<return type="void" />
+			<param index="0" name="platform" type="EditorExportPlatform" />
+			<description>
+				Adds an export platform to the available platforms list. The platform is initialized and becomes available for creating export presets.
```

Comment:
```suggestion
				Adds an export platform to the available platform list. The platform is initialized and becomes available for creating export presets.
```
No plural here in English, see [here](https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html#the-noun-adjunct)